### PR TITLE
Extend `fiberplane-charts` with theme prop

### PR DESCRIPTION
**Pull request by oscarvz**
# Description

Introduces a `chartTheme` prop to the `MetricsCard` component. It exposes a theme type with optional parameters which, when set, override the default theme set in `fiberplane-charts`.

As we initially extend `styled-component`'s `DefaultTheme` prop any consumer of the library using Styled Components will have their default theme extended, which is unwanted behavior.

TODO:
- [x] set values for the default theme
- [x] add additional annotations to theme values

~~Fixes FP-(issue)~~

# Checklist

~~- [ ] The changes have been tested to be backwards compatible.~~
~~- [ ] The OpenAPI schema and generated client have been updated.~~
~~- [ ] New models module has been added to api generator xtask~~
- [x] New types/fields are [well-documented and annotated](/CONTRIBUTING.md#adding-types-and-their-annotations).
- [x] The CHANGELOG is updated.

> Please link any related PRs in other repositories that depend on this:

After merging, please merge related PRs ASAP, so others don't get blocked.

### CODE REVIEW

**Comment by brettimus** [comment]
hey, i started work on the studio side, for when you can review: https://github.com/fiberplane/monofiber/pull/70

feel free to push changes directly to that branch!

File: fiberplane-charts/src/theme/types.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  /**
+   * The color to use for rendering events.
+   */
+  eventColor?: string;
+
+  /**
+   * The color to use for the gradient applied to an expendable event.
+   */
+  expendableGradientColor?: string;
```
Comment (brettimus) [inline fiberplane-charts/src/theme/types.ts]: nit - expandable instead of expendable

**Review by brettimus — APPROVED** [review]

**Comment by brettimus** [comment]
nice. this is a big improvement to the stylez plumbing inside charts, and resolves some previous hacks

File: fiberplane-charts/src/theme/types.ts (hunk 1 above)
Comment (oscarvz) [reply fiberplane-charts/src/theme/types.ts]: nice catch, will change!

**Pull request merged**